ansibleflow/app.py

```diff
@@ -50,13 +50,29 @@ def setup_argument_parser():
     )
 
     run_parser = subparsers.add_parser('run')
-    run_parser.add_argument('--env', type=str, default='default')
     run_parser.add_argument(
         'run_action',
-        nargs='*',
+        type=str,
         default=True,
         help=argparse.SUPPRESS
     )
+    run_parser.add_argument(
+            '--env',
+            type=str,
+            default='default')
+    run_parser.add_argument(
+            "-v",
+            "--verbosity",
+            action="count",
+            default=0,
+            help="increase output verbosity")
+    run_parser.add_argument(
+            '--file',
+            type=str,
+            default=None)
```

Comment:
When I first looked at this I was like "file"? I see now that you're referring to the project config. Perhaps we could name it something more descriptive then "file"?